one-ui/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.module.scss

```diff
@@ -0,0 +1,214 @@
+
+.mapContainer{
+    //padding: 40px 200px 100px 40px;
+    //margin-bottom: 100px;
+    height: 100%;
+    top: 8vh;
+    //box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+.header{
+    display:inline-flex;
+}
+.headerTitle{
+    font-size: 1.5em;
+    width: fit-content;
+    //border: 1px solid red;
+}
+
+.saveMessage{
+    //width: fit-content;
+    border: 1px solid #ff0000;
+    margin-left: 30em;
+}
+.noMessage{
+    padding: 2.25vh 0vh;
+}
+.parentContainer{
+    display: flex;
+    flex-direction: row;
+    padding-bottom: 2vh;
+    padding-top: 1vh;
+}
+
+.sourceContainer{
+    width: auto;
+    height: auto;
+    flex: 4;
+    display: flex;
+    flex-direction: column;
+    padding-right: 20px;
+    word-break: break-all;
+}
+
+.sourceTable {
+    & thead > tr > th {
+        background-color: #fafafa;
+        line-height: 2px;
+      };
+    & tbody > tr > td {
+        line-height: 2px;
+    }
+}
+.srcTableRows{
```

Comment:
Since "source" is used above, use "source" instead of "src" for consistency here. I don't think there's ever going to be a reason to abbreviate "source" or "entity".

Reply:
Done